packages/klevu-core/src/queries/index.ts

```diff
@@ -85,6 +85,10 @@ export type KlevuFetchFunctionReturnValue = {
    * Local memory cache products from previous results. Used by analytics when making page changes
    */
   previousResultRecords?: KlevuRecord[]
+  /**
+   * Recommendation title as set in KMC
+   */
+  title?: string
```

Comment:
There is no need to make changes to SDK. There is already a function get it. It's in the queryResult object.

Reply:
understood.

Reply:
i have updated to get it from the response of kmcRecommendations(), I couldnt find another way, let me know if there is.